rs/bitcoin/kyt/src/main.rs

```diff
@@ -13,6 +15,23 @@ async fn get_inputs(tx_id: String) -> Vec<String> {
 }
 
 #[ic_cdk::query]
+/// Return `Passed` if the given bitcion address passed the KYT check, or
+/// `Failed` otherwise.
+/// May throw error (trap) if the given address is malformed or not a mainnet address.
+async fn check_address(args: CheckAddressArgs) -> CheckAddressResponse {
```

Comment:
> I don't think check_address needs to be async so that an async runtime for those tests is actually not required)

That's a good point!

Reply:
> I don't think check_address needs to be async

You are right! I've removed the async.

But I think it is easier to test the trap/panic cases in an integration test. What do you think? @gregorydemay

Reply:
> But I think it is easier to test the trap/panic cases in an integration test. What do you think?

This is a minor point, but I'm not sure I see why it's easier. Generally, I would actually say that unit tests are way easier

1. No need for PocketIC or (any IC framework)
2. Much faster, hence can do for example `proptest`
3. (Somewhat) easy to assert that the panic occurred for the expected reason. For this you might need something like `expect_panic_with_message` ([here](https://github.com/dfinity/ic/blob/35bfcadd0f2a474057e42393917b8b3ac269627a/rs/ethereum/ledger-suite-orchestrator/src/state/test_fixtures.rs#L17))
4. This is a long shot: theoretically we could get test coverage report when not compiled to wasm for the pure Rust code.

But since we are talking about a few tests here, it's also fine by me if you prefer letting it as is.

Reply:
Those are some good reasons! Thanks for the clarification. But for this particular function, I think a prop test is likely overkill, so I'll just keep it as is.